static/login.css

```
@@ -0,0 +1,12 @@
+/*Body styling*/
+body {
+  background-image: url("../static/img1.jpg");
```

For any URL which references a file in the same directory as this CSS file, you can just write the name of the file and it should still work
```suggestion
  background-image: url("img1.jpg");
```